src/Components/News/NewsDetail.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { dateFormat } from '../../utils';
 
 const NewsDetail = ({ title, sysPublishDate, image, story }) => {
     return (
         <>
             <h3>{title}</h3>
-            <h5>{sysPublishDate}</h5>
+            <h5>{dateFormat(sysPublishDate)}</h5>
```

Comment:
I think we should create a React component to display date and that under the hood use the semantic html tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time